CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ All notable changes to this project will be documented in this file. See [Keep a
 CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.22.1 (2024-11-22)
```

Comment:
Note: you don't have to manually update the changelog. It's updated automatically based merge commit message (we use the [release-please](https://github.com/googleapis/release-please) action which bumps the version and generates the changelog based on conventional commits).